assets/css/colors/index.css

```diff
@@ -1,31 +1,31 @@
 .bg_orange{
-    background-color: #f1a177;
+    background-color: #f1a177 !important;
 }
 
 .textOrange{
-    color: #f1a177;
+    color: #f1a177 !important;
 }
 
 .bg_lightOrange{
-    background-color: #fadfd1;
+    background-color: #fadfd1 !important;
 }
 
 .textLightOrange{
-    color: #fadfd1;
+    color: #fadfd1 !important;
 }
 
 .bg_grey{
-    background-color: #c1b9b9;
+    background-color: #c1b9b9 !important;
```

Comment:
We are not going to use !important everywhere. Follow CSS pattern. Every component have ComponentName_componentChild class. Add color background color and everything there